src/components/PersonsList/PersonsList.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { memo } from 'react';
+import PersonItem from '../PersonItem/PersonItem';
+import { Person } from '../../types/Person';
+
+type Props = {
+  persons: Person[];
+  handleUserSelect: (user: Person) => void;
+};
+
+const PersonsList: React.FC<Props> = ({
```

Comment:
I suggest you merge this component with `PersonsDropDown` because it looks disproportional

Reply:
@Vayts If this component is no longer used, it should be removed.